internal/signerverifier/signerverifier.go

```diff
@@ -40,14 +41,16 @@ type keyVal struct {
 // keys. While this is called signerverifier, tuf.Key only supports public keys.
 //
 // Deprecated: Switch to upstream key loading APIs.
-func NewSignerVerifierFromTUFKey(key *tuf.Key) (dsse.SignerVerifier, error) {
+func NewSignerVerifierFromTUFKey(key *tuf.Key) (dsse.Verifier, error) {
```

Comment:
rename func too? I think we could undeprecate this tbh.

Reply:
No, we need another pass to handle this and update a bunch of tests to drop the legacy format. Let's do it then.